### ISSUE TITLE
CustomIdle doesn't work

### ISSUE THREAD

**Issue by RobinRMC** [RobinRMC/VencordPlus#43]
This issue also causes the friend list not to work.
<br>

Error message:
```
Uncaught (in promise) TypeError:
Vencord.Plugins.plugins.CustomIdle.getIdleTimeout(...) is not a function
```

**Comment by RobinRMC** [comment]
This issue will be fixed with the next update.

**Comment by RobinRMC** [comment]
The issue has been fixed with commit 4fb40b35f5aeafd0b9386f1fa5dc4501758cb3e9.
